src/components/shared/inputs/drop-down/drop-down.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useRef } from 'react';
+import DropDownMenu from './list/list';
+import DownArrow from '../../svg/down-arrow';
+
+export default function DropDown({
+  form = false,
+  setValue = '',
+  listItems = [],
+  attributes = {},
+  classNames = '',
+}) {
+  const [isToggled, setIsToggled] = useState(false);
+  const inputRef = useRef(null);
+
+  return (
+    <div className={classNames.container}>
+      <div className={classNames.inputGroup}>
+        {form && (
+          <label className={classNames.label}>Painting(s) interested in</label>
```

Comment:
I imagine you will want to pass this label in as a prop so you can re-use the dropdown later.